protected/humhub/modules/space/permissions/MembersManagePermission.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2015 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\space\permissions;
+
+use humhub\libs\BasePermission;
+use humhub\modules\space\models\Space;
+use Yii;
+
+/**
+ * Manage users to space without inviting them.
+ */
+class MembersManagePermission extends BasePermission
```

Comment:
We should use the global [humhub\modules\admin\permissions\ManageUsers permission](https://github.com/humhub/humhub/blob/51541424731113e25fffc402ae6b071d4f6d60f8/protected/humhub/modules/admin/permissions/ManageUsers.php). Otherwise each space admin would be allowed to force invite the whole communtiy, which is bad in case you allow normal users to create own spaces.